src/components/Home.js

```diff
@@ -0,0 +1,25 @@
+/* <eslint-disable max-len  */
```

Comment:
- [ ] OPTIONAL 
You have **disabled linter check** on your code. Please enable linter check on your code to ensure a cleaner syntax.